test/extended/testdata/build-secrets/test-docker-build.json

```diff
@@ -25,17 +25,17 @@
     },
     "strategy":{
       "type":"Docker",
-      "env": [
-        {
-          "name": "BUILD_LOGLEVEL",
-          "value": "5"
-        }
-      ],
```

Comment:
The positioning pre-dates my changes to the file: https://github.com/jupierce/origin/commits/e42a29f8cdec85ea26d98fcac328b7f850a671c6/test/extended/testdata/build-secrets/test-s2i-build.json

The one test I encountered in my travels relating to the build loglevel utilized --build-loglevel command line parameter instead of the buildconfig/env. A search of test/**/*.json for BUILD_LOGLEVEL finds these universally misplaced, so I don't think any unit test is checking it.